common/cross-profiles/aarch64.sh

```diff
@@ -7,3 +7,5 @@ XBPS_CXXFLAGS="$XBPS_CFLAGS"
 XBPS_CROSS_CFLAGS="-march=armv8-a"
 XBPS_CROSS_CXXFLAGS="$XBPS_CROSS_CFLAGS"
 XBPS_CROSS_FFLAGS=""
+RUSTFLAGS="--sysroot=${XBPS_CROSS_BASE}/usr"
```

Comment:
Hm, somehow this doesn't work (also for the other edited files). I can access RUSTFLAGS in a package (e.g. via `echo $RUSTFLAGS`), but it seems like cargo doesn't pick it up. If I do `export RUSTFLAGS="--sysroot...` in cargo's buildstyle environment it works - not quite sure what's different?

Reply:
isn't it unused anyway if you define it in `.cargo/config`?

Reply:
Yes, I define it in `.cargo/config` because it isn't picked up here - would be nice if we didn't have to edit `.cargo/config` for every rust package that doesn't use the cargo buildstyle.

Reply:
Export is missing, `common/xbps-src/shutils/common.sh` seems to be the place where most things get exported.

Reply:
Ah, so it has to be exported elsewhere, thought that'd be done automatically - thanks.